app/helpers/handlers.js

```diff
@@ -207,8 +207,10 @@ const getPage = async (question, request, h) => {
       if(getYarValue(request, 'applyingFor') === 'An impermeable cover only'){
         question.maybeEligibleContent.isimpermeablecoveronly = true
         question.nextUrl = `${urlPrefix}/project-type`
-      }else if(getYarValue(request, 'applyingFor') === 'Building a new store, replacing or expanding an existing store' && getYarValue(request, 'fitForPurpose') === 'No'){
-        question.maybeEligibleContent.isimpermeablecoveronly = false
+      }else{
+        if(getYarValue(request, 'applyingFor') === 'Building a new store, replacing or expanding an existing store' && getYarValue(request, 'fitForPurpose') === 'No'){
```

Comment:
this isn't what i meant xD just put

} else {
question.maybeEligibleContent.isimpermeablecoveronly = false
}

Dont have the if statement in there at all :) No need to if you only have 2 possible scenarios